Week-7/LinkedList(7.1-7.3)/index.js

```diff
@@ -0,0 +1,127 @@
+// Question 7.1,7.2 and 7.3 covered in this file
```

Comment:
Instead of this, could have created a separate file for Creating/Traversing/Adding nodes in a linked list and separate out all questions in different files.